tools/shared/DotNetMuxer.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+// Copy of code from https://github.com/aspnet/DotNetTools/blob/047e4a8533bbcdf22831c40dd9d9aaf0c80d1953/shared/DotNetMuxer.cs
```

Comment:
I could reference [CLI.Utils](https://www.nuget.org/packages/Microsoft.DotNet.Cli.Utils) here instead and use their version of this type. How about that?